Cython/Compiler/Tests/TestCmdLine.py

```diff
@@ -98,21 +99,62 @@ def test_options_with_values(self):
         self.assertTrue(options.gdb_debug)
         self.assertEqual(options.output_dir, '/gdb/outdir')
 
+    def test_module_name(self):
+        options, sources = parse_command_line([
+            'source.pyx'
+        ])
+        self.assertEqual(options.module_name, None)
+        self.check_default_global_options()
+        self.check_default_options(options)
```

Comment:
These functions don't [exist](https://github.com/cython/cython/blob/0.29.x/Cython/Compiler/Tests/TestCmdLine.py) on the 0.29 branch. Would you like me to add the [functions](https://github.com/cython/cython/blob/master/Cython/Compiler/Tests/TestCmdLine.py#L23-L31), or remove the calls here?

Reply:
Probably add the calls? They look fairly self-contained to me so it shouldn't involve back-porting a lot of code.